L1Trigger/Phase2L1ParticleFlow/python/L1NNTauProducer_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+L1NNTauProducer = cms.EDProducer("L1NNTauProducer",
+                                 seedpt          = cms.double(20),
+                                 conesize        = cms.double(0.4),
+                                 tausize         = cms.double(0.1),
+                                 maxtaus         = cms.int32(5),
+                                 nparticles      = cms.int32(10),
+                                 L1PFObjects     = cms.InputTag("L1PFProducer","l1pfCandidates"),
+                                 NNFileName      = cms.string("L1Trigger/Phase2L1ParticleFlow/data/tau_3layer.pb")
+                                 )
+
+
+L1NNTauProducerPuppi = cms.EDProducer("L1NNTauProducer",
```

Comment:
the only difference here is the `NNFileName`, so just clone the above and change that

Reply:
done